app/api/datastore/entities/maat/application.rb

```diff
@@ -2,13 +2,48 @@ module Datastore
   module Entities
     module Maat
       class Application < Grape::Entity
+        expose :id
+        expose :schema_version
         expose :reference
-        expose :applicant
+        expose :submitted_at, format_with: :iso8601
+        expose :date_stamp
+        expose :client_details
+        expose :provider_details
+        expose :case_details
+        expose :interests_of_justice
+        expose :ioj_passport
 
         private
 
-        def applicant
-          object.application&.dig('client_details', 'applicant')
+        def client_details
+          object.application&.dig('client_details')
+        end
+
+        def schema_version
+          1.0
+        end
+
+        def date_stamp
+          object.application&.dig('date_stamp')
+        end
+
+        def provider_details
+          object.application&.dig('provider_details')
+        end
+
+        def case_details
+          case_details = object.application&.dig('case_details')
+          case_details['offence_class'] = nil
```

Comment:
I guess this is a holding value until the algorithm for the offence class mapping has been defined?